multisig/proposals/arbitrumone/vip-013/index.ts

```diff
@@ -0,0 +1,232 @@
+    },
+    {
+      target: arbitrumone.POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [
+        [
+          VwstETH,
+          parseUnits("0.93", 18),
+          parseUnits("0.95", 18),
+          parseUnits("2", 18),
+          vTokenReceiver,
+          parseUnits("8000", 18),
+          parseUnits("800", 18),
+        ],
+      ],
+    },
+    {
+      target: arbitrumone.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          weETH,
+          [
+            weETH_ONE_JUMP_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: arbitrumone.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[weETH, CHAINLINK_weETH_FEED, STALE_PERIOD_26H]],
+    },
+    {
+      target: arbitrumone.VTREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [weETH, parseUnits("2", 18), arbitrumone.GUARDIAN],
+    },
+    {
+      target: weETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, 0],
+    },
+    {
+      target: weETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, parseUnits("2", 18)],
+    },
+    {
+      target: VweETH,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["86400"],
+    },
+    {
+      target: VweETH,
+      signature: "setProtocolShareReserve(address)",
+      params: [PSR],
+    },
+    {
+      target: VweETH,
+      signature: "setProtocolSeizeShare(uint256)",
+      params: [parseUnits("0.01", 18)],
+    },
+    {
+      target: arbitrumone.POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [
+        [
+          VweETH,
+          parseUnits("0.93", 18),
+          parseUnits("0.95", 18),
+          parseUnits("2", 18),
+          vTokenReceiver,
+          parseUnits("4600", 18),
+          parseUnits("2300", 18),
+        ],
+      ],
+    },
+    {
+      target: arbitrumone.VTREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [WETH, parseUnits("1.9678", 18), arbitrumone.GUARDIAN],
+    },
+    {
+      target: WETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, "0"],
+    },
+    {
+      target: WETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, parseUnits("1.9678", 18)],
+    },
+    {
+      target: VWETH,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["86400"],
+    },
+    {
+      target: VWETH,
+      signature: "setProtocolShareReserve(address)",
+      params: [PSR],
+    },
```

Comment:
This command is not needed, because the vWETH contract already has the right PSR contract configured: [0xF9263eaF7eB50815194f26aCcAB6765820B13D41](https://arbiscan.io/address/0xF9263eaF7eB50815194f26aCcAB6765820B13D41)

Reply:
Added here, https://github.com/VenusProtocol/vips/commit/eb462e035c9d57ad65cdb41577db75d3fc1ee62b